deployer/src/config_manager.py

```diff
@@ -0,0 +1,86 @@
+import algoliasearch
+import json
+
+from . import algolia_helper
+from . import snippeter
+from . import emails
+from . import helpers
+from .dict_differ import DictDiffer
+from . import fetchers
+
+from helpdesk_helper import add_note, get_conversation, get_emails_from_conversation, get_conversation_url_from_cuid
+
+
+class ConfigManager:
+    differ = None
+
+    def __init__(self):
+        self.ref_configs = fetchers.get_configs_from_repos()
+        self.actual_configs, self.inverted_actual_configs, self.crawler_ids = fetchers.get_configs_from_website()
+
+        self.differ = DictDiffer(self.ref_configs, self.actual_configs)
+
+    def get_added(self):
+        return self.differ.added()
+
+    def get_removed(self):
+        return self.differ.removed()
+
+    def get_changed(self):
+        return self.differ.changed()
+
+    def add_config(self, config_name):
+        key = algolia_helper.add_docsearch_key(config_name)
+        print(config_name + ' (' + key + ')')
+        helpers.make_request('/', 'POST', {'configuration': json.dumps(config_name, separators=(',', ': '))})
```

Comment:
this is not correct @maxiloc 
should be the full config `json.dumps(ref_configs[config],`